### PR TITLE
Document all explicit permissions for access group

### PR DESCRIPTION
**Pull request by plamenbotev**

### CODE REVIEW

File: version1/04 - Web services.md [hunk 1]
```diff
@@ -1800,6 +1800,63 @@ Content-type: application/json
...
+     ],
+     "hasMore": true
+}
+```
+
+###### Details
+
+- `results`
+  - contains the (potentially empty) objects with additional metadata and explicit permissions for the specified group
+  - at most 100 objets per page can be retrieved
```
Comment (ivaylomitrev) [inline version1/04 - Web services.md:1882]: I think we must describe what values `discriminator` and `listtype` can take.

File: version1/04 - Web services.md [hunk 2]
```diff
@@ -1800,6 +1800,63 @@ Content-type: application/json
 - **servicePermissions**
   - the group service permissions
 
+### Get all objects explicit permissions for access groups by ID
```
Comment (ivaylomitrev) [inline version1/04 - Web services.md]:
```suggestion
### Get all permissions assigned to access group
```
I think this is more readable 🔝

File: version1/04 - Web services.md (hunk 1 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md:1882]: We cam also mention what fields are expected for each discriminator type (i.e. when `objectId` is expected, when `listType` and when `listType` _and_ `code`)

File: version1/04 - Web services.md [hunk 3]
```diff
@@ -1800,6 +1800,63 @@ Content-type: application/json
...
+         },
+         {
+             "accessGroupId": int,
+             "explicitPermissions": [string],
+             "discriminator": string,
+             "listType": string,
+             "code": string
+         }
+     ],
+     "hasMore": true
```
Comment (ivaylomitrev) [inline version1/04 - Web services.md]:
```suggestion
     "hasMore": boolean
```

We must also describe what this field is in the `Details` section below. You can check out earlier occurrences of it in the document.

**Comment by ivaylomitrev** [comment]
No other comments.

File: version1/04 - Web services.md (hunk 1 above)
Comment (plamenbotev) [reply version1/04 - Web services.md:1882]: Additional information by the fields and the different types of discriminators and supported code lists and code list values was added. :)

File: version1/04 - Web services.md (hunk 2 above)
Comment (plamenbotev) [reply version1/04 - Web services.md]: Changed. :)

File: version1/04 - Web services.md (hunk 3 above)
Comment (plamenbotev) [reply version1/04 - Web services.md]: Additional information was added. :)

**Comment by ivaylomitrev** [comment]
I think you just need a rebase on develop now, because my last commits seem to be present in your PR too (haven't checked the tree to see why). Ready to approve after that :)

**Comment by plamenbotev** [comment]
@ivaylomitrev The branch was rebased and squashed. :)

File: version1/04 - Web services.md (hunk 2 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: Are you sure? :D

**Review by ivaylomitrev — APPROVED** [review]

**Pull request merged**